kodi_addon_checker/__main__.py

```diff
@@ -11,6 +11,17 @@
 ROOT_URL = "http://mirrors.kodi.tv/addons/{branch}/addons.xml"
 
 
+def _all_repo_addons():
+    branches = ['gotham', 'helix', 'isengard', 'jarvis', 'krypton', 'leia']
+    repo_addons = {}
+
+    for branch in branches:
+        branch_url = ROOT_URL.format(branch=branch)
+        repo_addons[branch] = check_addon._get_addons(branch_url)
```

Comment:
Please add some logging here and portentially in _get_addons. Locally it seems to get stuck on downloading or well take forever. Might be my connection. But logging something might not hurt. 
Do normal logs, not our report stuff :)

Reply:
Or well don't do that as it will likely show up in the travis log too and mess up our reports.
Kinda complicated to have both.

Reply:
maybe load the addons.xml.gz version, it is much smaller

Reply:
Then we'll have to unzip it right ?

Reply:
yes we can do that. we'll have to add something like this:
```python
  import gzip
  with gzip.open('file.tar.gz', 'rb') as f:
  file_content = f.read()
```
Then add the `file_content` to the dictionary.
Should I do it ?

Reply:
Absolutely should be an awesome improvement. I'm far more concerned for a file transfer to break then a local zip operation not working :)